docs/element-definitions.md

```diff
@@ -2337,6 +2337,44 @@ Also, please note this example is incomplete for the sake of keeping it short.
 }
 ```
 
+## Authentication and Authorization Scheme Requirement ([Array][])
+
+This represents a group of schemes where each of them MUST be satisfied when making a transaction.
+
+#### Template
+
+- `element`: `authSchemeRequirement`
+- `content`
+    - ([Auth Scheme][])
+
+#### Example
+
+Assuming we have a custom basic authentication scheme defined as `Custom Basic Auth` and a custom token authentication scheme defined as `Custom Token Auth`, the following when used on an HTTP transaction within a resource implies that both authentication schemes MUST be satisfied. Please note this example is incomplete for the sake of keeping it short.
```

Comment:
For me, something like the following would be more readable, probably I end up here coming from the Auth Scheme definition anyway.

* Assuming a custom basic authentication scheme defined as `Custom Basic Auth` and a custom token authentication scheme defined as `Custom Token Auth`, the example below illustrates an HTTP transaction requiring both authentication schemes.